docs/TESTING_WITH_SANITIZERS.md

```diff
@@ -13,6 +13,20 @@ This is mostly done automagically by various CI systems for each PR, but you may
 want to do it locally as well. The process slightly varies depending on the
 compiler you want to use and which part of the test suite you want to run.
 
+## mkosi
+
+To build with sanitizers in mkosi, create a file 20-local.conf in mkosi.default.d/ and add the following
+contents:
+
+```
+[Content]
+Environment=SANITIZERS=address,undefined
+```
+
+The value of `SANITIZERS` is passed directly to meson's `b_sanitize` option, so you can set it to any value
+supported by `b_sanitize`. See https://mesonbuild.com/Builtin-options.html#base-options for a list of
```

Comment:
It think it's a bit misleading in the sense that for example it isn't possible to just pass `memory` here and expect it to work because it would require `clang`, the dependencies built with `MSan`, `MSAN_OPTIONS`, `LD_LIBRARY_PATH` sprinkled here and there and `libc` more or less compatible with `MSan` and so on. I think it would be fair to say that `address` and `undefined` (shipped with `gcc`) is what it can work with and what is supported at this point.

Reply:
Updated to mention only the gcc sanitizers are supported